app/page.tsx

```
@@ -21,6 +24,23 @@ async function fetchRankingData(genre: string = 'all', period: string = '24h', t
   popularTags?: string[]
 }> {
   
+  // 0. Cloudflare KVから読み取りを試みる（タグなしの場合のみ）
+  if (!tag) {
+    try {
+      const cfData = await getGenreRanking(genre, period as RankingPeriod)
+      if (cfData && cfData.items && cfData.items.length > 0) {
+        // NGフィルタリングを適用
+        const filteredData = await filterRankingData({
+          items: cfData.items.slice(0, 100), // 最初の100件のみ
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)